sklearn/metrics/cluster/_unsupervised.py

```diff
@@ -122,7 +123,7 @@ def _silhouette_reduce(D_chunk, start, labels, label_freqs):
 
     Parameters
     ----------
-    D_chunk : array-like of shape (n_chunk_samples, n_samples)
+    D_chunk : {array-like, sparse CSR matrix} of shape (n_chunk_samples, n_samples)
         Precomputed distances for a chunk.
```

Comment:
```suggestion
        Precomputed distances for a chunk. When providing a sparse matrix, a
        CSR format should be provided to avoid an additional copy.
```